torch/_inductor/codegen/triton_utils.py

```diff
@@ -37,37 +37,36 @@ def signature_to_meta(signature, *, size_dtype: str):
 def config_of(args):
     from ..compile_fx import ALIGNMENT
 
-    def is_aligned(x):
-        if isinstance(x, TensorArg):
-            return x.buffer not in V.graph.unaligned_buffers
-        if isinstance(x, SizeArg):
-            # TODO(voz): These are kinda redundant, if we can solve out statically_known_multiple_of with
-            # _maybe_evaluate_static...
-            if x.name.startswith("load_seed_offset"):
-                return False
-            else:
-                return V.graph.sizevars.statically_known_multiple_of(x.expr, ALIGNMENT)
-        raise NotImplementedError(f"unhandled {type(x)}: {x}")
-
-    def is_aligned_8(x):
+    def is_aligned(x, alignment, include_tensor):
         """
         Roughly follow triton code here:
         https://github.com/openai/triton/blob/5282ed890d453e10b9ee30076ef89115dd197761/python/triton/runtime/jit.py#L208-L222
         """
         if isinstance(x, TensorArg):
-            return False
+            if include_tensor:
+                return x.buffer not in V.graph.unaligned_buffers
+            else:
+                return False
         if isinstance(x, SizeArg):
             # TODO(voz): These are kinda redundant, if we can solve out statically_known_multiple_of with
             # _maybe_evaluate_static...
             if x.name.startswith("load_seed_offset"):
                 return False
             else:
-                return V.graph.sizevars.statically_known_multiple_of(x.expr, 8)
+                return V.graph.sizevars.statically_known_multiple_of(x.expr, alignment)
         raise NotImplementedError(f"unhandled {type(x)}: {x}")
 
     if config.triton.divisible_by_16:
-        divisible_by_16 = tuple(i for i, arg in enumerate(args) if is_aligned(arg))
+        divisible_by_16 = tuple(
+            i
+            for i, arg in enumerate(args)
+            if is_aligned(arg, alignment=ALIGNMENT, include_tensor=True)
```

Comment:
```suggestion
            if is_aligned(arg, alignment=16, include_tensor=True)
```
given the name of the attribute, I think hardcoding this is fine.